sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5404,36 +5388,34 @@ pi_result _pi_event::cleanup(pi_queue LockedQueue) {
       // Event has been signalled: If the fence for the associated command list
       // is signalled, then reset the fence and command list and add them to the
       // available list for reuse in PI calls.
-      if (Queue->RefCount > 0) {
```

Comment:
Why this is not needed any more (or was redundant)?

Reply:
If Queue->RefCount == 0 we are not allowed to use Queue object, it can be deleted at any point as soon as ref count reaches 0. So, this check doesn't make sense.

Reply:
I agree, just wondered what it was solving when it was added.